edn/src/types.rs

```diff
@@ -143,6 +162,16 @@ impl Value {
     def_as!(as_list, List, LinkedList<Value>);
     def_as!(as_set, Set, BTreeSet<Value>);
     def_as!(as_map, Map, BTreeMap<Value, Value>);
+
+    pub fn from_bigint(src: &str) -> Result<Value, NewValueError> {
```

Comment:
AFAIK Result is always preferable to Option when errors *can* be specified. However, if they *can* be, doesn't mean that we necessarily have to. Do you see the potential NewValueError sum type to grow in the future? Is it worth allowing consumers to know exactly why creating an edn::Value fails?